certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW

```diff
@@ -346,17 +348,21 @@ DEFINE
  SXFER3EXPIREDATE          = DATE ARRAY(99)
  SXFER3LOC                 = NUMBER ARRAY(99)
  SXFER3SHAREID             = CHARACTER(4) ARRAY(99)
-
  STRANSFERCOUNTMAX         = 99
+
+[* Added 06/06/24 *]
+ PARAMINSTALLDATE          = DATE
+ PARAMDEBUGMODE            = NUMBER
+ PARAMDEBUGMODEFOUND       = NUMBER
 END [DEFINE]
 
 SETUP
  INCLUDEPROGRAMINFO=TRUE
- BNOLASTMODDATE='03/14/24'
+ BNOLASTMODDATE='06/06/24'
  BNOLASTMODTIME="12:00 PT"
  BNOPROGRAMNAME="BANNO.CDRENEW.V1.POW"
  BNOCFGFILENAME = "BANNO.CDRENEW.V1.CFG"
- BNOPROGRAMVERSION="1.4.1"
+ BNOPROGRAMVERSION="1.6.0"
```

Comment:
@rrobison44 meant this to actually be a 1.5.0 instead of a 1.6.0.  But I can see considering this as a fix instead of a feature bump - even though nothing is broken per-se that this is fixing.  I'll change it to a 1.4.2.

Reply:
@rrobison44 Actually.... it would be 1.5.1 as the last set of mods to the program were not correctly reflected in the setting of the BNOPROGRAMVERSION variable to 1.5.0 as it should have been:
```
**    Ver. 1.5.0  03/14/24 TKainz - Banno
**                Added parameter setting to delete/expire any transfer type 3
**                records pointing to the targeted certificate upon the member
**                submitting their selection.
```

Reply:
Yeah, I guess just be consistent with the version bumps you did for the other programs for which you added this param.